### ISSUE TITLE
image input 

### ISSUE THREAD

**Issue by Dpw506** [rishikksh20/SiT-pytorch#2]
Hi, I am very glad to discuss with you about Transformer. I would like to ask you how to input data into the network if it is a image.I want to see the effect of the specific picture here, and I hope to receive your reply as soon as possible.
In addition, I tried to change the image into four dimensions by myself, and then presented the results, but the results were garbled. Have you ever tried to send the pictures into the SIT model? Thank you so much!

**Comment by rishikksh20** [comment]
* Load images
* Convert them to tensor or dataloader if you are training 
* Make sure the dimension of image is 3 and 4th dimension is batch dim
* And then just pass that tensor to SiT model

**Comment by Dpw506** [comment]
Ok, Thank you for your reply!
